lib/Core/EpisodeAction/EpisodeActionSaver.php

```diff
@@ -110,7 +107,7 @@ private function hydrateEpisodeActionEntity(EpisodeAction $episodeAction, string
 		$episodeActionEntity->setPosition($episodeAction->getPosition());
 		$episodeActionEntity->setStarted($episodeAction->getStarted());
 		$episodeActionEntity->setTotal($episodeAction->getTotal());
-		$episodeActionEntity->setTimestamp($this->convertTimestampToDbDateTimeString($episodeAction->getTimestamp()));
+		$episodeActionEntity->setTimestamp($this->ensureCorrectDateTimeFormat($episodeAction->getTimestamp()));
```

Comment:
we now use `timestamp_epoch`. `timestamp` is deprecated.

Reply:
Should be fixed with the rebase. I don't fully understand git rebase, but now everything seems to be fine.